desktop-app/app/main.dev.js

```diff
@@ -212,6 +220,7 @@ const createWindow = async () => {
       nodeIntegrationInWorker: true,
       webviewTag: true,
       enableRemoteModule: true,
+      webSecurity: false,
```

Comment:
@esprush Any specific reason for adding this?

Reply:
added for experimental purpose. removed this.